src/Documentation/UserManual/DeviceAtracsys.dox

```diff
@@ -8,15 +8,14 @@
 
 \section AtracsysSupportedPlatforms Supported platforms
 
-- \ref PackageWin32
```

Comment:
I removed it from the doc because we have not released a win32 version of our SDK for more than 3 years and to our knowledge, none of our current clients use a win32 platform. That said, I would understand if you wanted to leave it in for platforms using much older versions of our API, but we have not supported those for a long time.

Reply:
Good point, if there's no recent win32 SDK support for Atracsys devices then let's remove the PackageWin32 option